notebooks/00_quick_start/sar_movielens_with_aml.ipynb

```diff
@@ -0,0 +1,471 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Microsoft Corporation. All rights reserved.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train SAR Single Node on MovieLens with Azure Machine Learning (Python, CPU)\n",
+    "\n",
+    "This notebook provides an exmaple of how to train SAR on remote compute resources. Details of SAR algorithm can be found in [SAR Python CPU Movielens](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/sar_movielens.ipynb) notebook. \n",
+    "\n",
+    "This notebook showcases what are the changes required to run local script on AzureML targets. AzureML comes with auto-scaling and gpu options and can greatly accelerate model training. There're a few more notebooks in `04_model_select_and_optimize` folder that shows how to use AzureML to do hyperparameter tuning. \n",
+    "\n",
+    "### Prerequisite\n",
+    "This notebook requires an AzureML workspace to be set up. Please follow [this notebook](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb) on how to create an AzureML workspace."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0 Setup environment\n",
+    "### Get workspace and create experiment\n",
+    "The workspace was created in configuration notebook and can be loaded using `Workspace.from_config()`.\n",
+    "\n",
+    "The DSVM created in configuration notebook is called `cpucluster`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Found the config file in: /data/home/cathack/notebooks/Recommenders/notebooks/aml_config/config.json\n",
+      "setup-ws\n",
+      "jingywa-test\n",
+      "eastus2\n",
+      "b8c23406-f9b5-4ccb-8a65-a8cb5dcd6a5a\n"
+     ]
+    }
+   ],
+   "source": [
+    "# set the environment path to find Recommenders\n",
+    "import sys\n",
+    "sys.path.append(\"../../\")\n",
+    "from reco_utils.dataset import movielens\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "import azureml\n",
+    "from azureml.core import Workspace, Run, Experiment\n",
+    "from azureml.train.estimator import Estimator\n",
+    "from azureml.widgets import RunDetails\n",
+    "\n",
+    "# load workspace configuration from the config.json file in the current folder.\n",
+    "ws = Workspace.from_config()\n",
+    "print(ws.name, ws.resource_group, ws.location, ws.subscription_id, sep = '\\n')\n",
+    "# create experiment\n",
+    "experiment_name = 'movielens-sar'\n",
+    "exp = Experiment(workspace=ws, name=experiment_name)\n",
+    "# attach amlcompute\n",
```

Comment:
Please explicitly show how to create the compute and do not reference the configuration notebook. Also, above you say it's DSVM and here you say amlcompute

Reply:
@ateste is working on an azure setup for CVBP repo, please sync with her to be aligned on how to show the workspace setup

Reply:
Yes, I just submitted a [PR](https://msdata.visualstudio.com/Vienna/_git/AzureMlCli/pullrequest/197765?_a=overview) to the SDK repo.
If accepted, it will allow the user to retrieve an already exiting/create a new workspace, with just one line of code: ws = Workspace.setup().

Reply:
Added a cell using %load create_workspace.py